modules/tensor_mechanics/src/materials/ComputeAxisymmetric1DSmallStrain.C

```diff
@@ -45,8 +60,13 @@ ComputeAxisymmetric1DSmallStrain::initialSetup()
 Real
 ComputeAxisymmetric1DSmallStrain::computeStrainYY()
 {
-  if (_scalar_out_of_plane_strain_coupled)
-    return _scalar_out_of_plane_strain[0];
+  if (_has_scalar_out_of_plane_strain)
+  {
+    const unsigned int scalar_var_id =
```

Comment:
This code shows up in a lot of places.  To clean this up, it might be good to have a method that we call that does this.  Maybe we could have a base class that these classes all derive from that handles some of this?  Let's discuss.